Source/main.cpp

```diff
@@ -18,50 +18,60 @@
   */
 /* USER CODE END Header */
 /* Includes ------------------------------------------------------------------*/
+#include <stdint.h>
+#include <time.h>
 #include "main.h"
-#include "adc.h"
-
 #include "hydrophones.h"
-#include <stdint.h>
-
 #include "stm32f7xx_hal.h"
+#include "adc.h"              /* Somehow throws an error */
```

Comment:
Critical error or just a warning? Might be nice to fix before merging if it is critical 😅 (or remove comment if it doesn't throw an error anymore)